modules/nw-sriov-configuring-multiple-nodes.adoc

```diff
@@ -0,0 +1,171 @@
++
+.Example `sriov-nw-pool.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: SriovNetworkPoolConfig
+metadata:
+  name: pool-1 <1>
+  namespace: openshift-sriov-network-operator <2>
+spec:
+  maxUnavailable: 2 <3>
+  nodeSelector: <4>
+    matchLabels:
+      node-role.kubernetes.io/worker: ""
+----
+<1> Specify the name of the `SriovNetworkPoolConfig` object.
+<2> Specify namespace where the SR-IOV Network Operator is installed.
+<3> Specify an integer number, or percentage value, for nodes that can be unavailable in the pool during an update. For example, if you have 10 nodes and you set the maximum unavailable to 2, then only 2 nodes can be drained in parallel at any time, leaving 8 nodes for handling workloads.
+<4> Specify the nodes to add the pool by using the node selector. This example adds all nodes with the `worker` role to the pool.
+
+.. Create the `SriovNetworkPoolConfig` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-nw-pool.yaml
+----
+
+. Create the `sriov-test` namespace by running the following comand:
++
+[source,terminal]
+----
+$ oc create namespace sriov-test
+----
+
+. Create a `SriovNetworkNodePolicy` resource:
+
+..  Create a YAML file that defines the `SriovNetworkNodePolicy` resource:
++
+.Example `sriov-node-policy.yaml` file
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriov-nic-1
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice
+  nicSelector:
+    pfNames: ["ens1"]
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+  numVfs: 5
+  priority: 99
+  resourceName: sriov_nic_1
+----
+
+.. Create the `SriovNetworkNodePolicy` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-node-policy.yaml
+----
+
+. Create a `SriovNetwork` resource:
+
+.. Create a YAML file that defines the `SriovNetwork` resource:
++
+.Example `sriov-network.yaml` file
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-nic-1
+  namespace: openshift-sriov-network-operator
+spec:
+  linkState: auto
+  networkNamespace: sriov-test
+  resourceName: sriov_nic_1
+  capabilities: '{ "mac": true, "ips": true }'
+  ipam: '{ "type": "static" }'
+----
+
+.. Create the `SriovNetwork` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-network.yaml
+----
+
+.Verification 
+
+* View the node pool you created by running the following command:
++
+[source,terminal]
+----
+$ oc get sriovNetworkpoolConfig -n openshift-sriov-network-operator
+----
++
+.Example output
+[source,terminal]
+----
+NAME     AGE
+pool-1   67s <1>
+----
+<1> In this example, `pool-1` contains all the nodes with the `worker` role.
+
+To demonstrate the node draining process using the example scenario from the above procedure, complete the following steps:
```

Comment:
```suggestion
To demonstrate the node draining process by using the example scenario from the previous procedure, complete the following steps:
```
Should this be formatted as a bulleted item like the bulleted item above it on line 124?